src/js/services/forms.js

```diff
@@ -310,8 +310,10 @@ export default App.extend({
       }).catch(({ responseData }) => {
         /* istanbul ignore next: Don't handle non-API errors */
         if (!responseData) return;
+
+        this.trigger('error', responseData.errors);
+
         const errors = map(responseData.errors, 'detail');
-        this.trigger('error', errors);
```

Comment:
we're not currently using the map anywhere I assume

Reply:
Not that I could I find... but I'll double check that.

I think it's only used for the `channel.request('send', 'form:errors', errors);` in the `/js/formapp.js` file [here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/formapp.js#L105-L115).